src/components/Accordion.js

```diff
@@ -100,16 +103,16 @@ const allMenuItemsAction = button => {
   const isMenuOpen =
     button.textContent.toLowerCase().trim() === "open all" ? true : false;
 
-  var body = button.closest(".dg_accordion");
-  var menuItems = body.getElementsByClassName("multi-collapse");
-  var totalCollapsed = body.getElementsByClassName("show");
+  var bodyElm = button.closest(".dg_accordion");
+  var menuItems = bodyElm.getElementsByClassName("multi-collapse");
```

Comment:
`getElementsByClassName` returns an [HTMLCollection](https://developer.mozilla.org/en-US/docs/Web/API/HTMLCollection) which is similar to nodelist, so i think `Elms` suffix makes sense here and on line 108.

Reply:
So the only time we are making a variable plural like var animals = [dog, cat] is if it contains say, string or integer types and if its an array of objects then its var animalElms = getElementsByClassName("dog")?

Reply:
I think what you are saying is correct, but I'm slightly confused. I think an array of anything we can name like 'dogs' or 'animals'. I think the document.get... or document.query is the exception? Does that make sense?

Reply:
Yeah I think we are on the same page. I was just debating with myself if I agreed with HTMLCollection as an Elms instead of just a plural (dogElms vs dogs). But since technically an HTMLCollection isn't an array this makes sense. Lots to keep track of.....hahaha